src/components/Button/Button.tsx

```diff
@@ -1,16 +1,12 @@
 import clsx from 'clsx';
 import React, { MouseEventHandler, ReactNode } from 'react';
 import styles from './Button.module.css';
-import { Icon } from '../Icon/Icon';
+
 export interface Props {
   /**
    * Button reference
    */
   buttonRef?: any;
-  /**
-   * Toggle between full width button (true) and default button width
-   */
-  block?: boolean;
```

Comment:
For `block` and `fullWidth`, it's not clear to me why we would need both (I only see `full-width` in the css and the comment calls it a "block button"). More importantly, I haven't seen anything in the designs to suggest this is an option we'd want to support. However, I'm open to being convinced that we should keep it because it's a common paradigm in web design.

Reply:
@dierat This is a good thing to flag with the broader group! In our experience over the years, we've never encountered a system that didn't utilize a `fullWidth` button. So my gut says that it's good to keep around, but we can also remove it until there's a tangible use case for it. 

As far as naming goes, we'd prefer `fullWidth` over `block` as it's more descriptive. We've worked with designers who interpret a "block" button as "filled" (read: primary), so `fullWidth` is more explicit of its purpose.